New python file : exceptions.py, Better exception handling

Mazzya opened the pull request:
It is advisable to have a separate file that contains the different exceptions, the objective is to be able to add and modify exceptions in a simple and fast way. I have created a file called exceptions.py, this file contains the following exceptions:
* NotFound404
* NoResultFound
* TooManyRequests

To use one of the exceptions, it is imported directly from the file. Look at the code to understand it better.
When a new exception is added it should be directly in this new file called exceptions.py

Mazzya committed:
Creating the exception file : exceptions.py

This file contains all the exceptions of the module, when a new exception must be added, it must be directly in the exceptions.py file.

DriftAsimov commented:
Great!

Just change `from exceptions import Exception` to `from animec.exceptions import Exception`

Mazzya committed:
import exception updated

```from exceptions import Exception``` -> ```from animec.exceptions import Exception```

Mazzya commented:
Done 👍🏼
